### ISSUE TITLE
Steam game dll copying not working

### ISSUE THREAD

**Issue by Redn2000** [LibreVR/Revive#1524]
After going through the wiki and setup instructions, I tried copying the required Revive dll files to the game I want to play with Rift only support from Steam. 

Once they were added, I tried to launch Revive which didn't show the game. The same happens when trying to launch it from within SteamVR. It only allows me to play it in Virtual Desktop mode.

I've tried this both with the Oculus program with Revive running or just Revive alone. Is there something I may have missed or a tutorial that can better explain what else I need to do?

**Comment by DaRkL3AD3R** [comment]
I'm having this issue too with Revive 2.1.x and Dirt Rally Steam. I have a bundle of old dlls from an older version of Revive when patching games was still present as a feature, and it works, but it's buggy and crashes randomly on me which I think is Revive's fault.

**Comment by CrossVR** [comment]
Patching is still available, check out the wiki.

**Comment by DaRkL3AD3R** [comment]
Unfortunately it doesn't work. I followed the instructions to the letter. Dirt Rally is a 32 bit app. I tried copying the exact files the wiki says, I tried copying the 32 bit ones (LibRevive32.dll + openvr_api32.dll) and nothing I do makes it work. The only solution is grabbing my old backup dll files from a much older version of Revive. 1.41 according to the LibRevive32.dll file details but that doesn't sound right.

**Comment by CrossVR** [comment]
Did you also copy `xinput1_3.dll`?

**Comment by DaRkL3AD3R** [comment]
Yes, I copied all pertinent and proper files. Doesn't engage VR at all, just loads as normal.

**Comment by Vertimyst** [comment]
I'm having the same issue.  It just loads as though I don't have a VR headset connected.

**Comment by TaliKert** [comment]
Had the same issue. Downgrade to 1.8.1, patch drt.exe and launch with SteamVR running.
